From fff39b1dc4ade97157426041a60c21584abd5a1a Mon Sep 17 00:00:00 2001
From: hwzjyggsddu <zhujiayuan@huawei.com>
Date: Tue, 23 Mar 2021 06:48:03 +0000
Subject: [PATCH] fix: extract tool files and some dynamic library to new added
 sub-package utils.

---
 libdb.spec | 18 +++++++++++++-----
 1 file changed, 13 insertions(+), 5 deletions(-)

diff --git a/libdb.spec b/libdb.spec
index 3c130a0..3139b2b 100644
--- a/libdb.spec
+++ b/libdb.spec
@@ -61,9 +61,17 @@ transactional database engine with proven reliability and availability.
 Berkeley DB comes three versions: Berkeley DB, Berkeley DB Java 
 Edition, and Berkeley DB XML.
 
+%package        utils
+Summary:        utils for libdb
+Requires:       %{name} = %{version}-%{release}
+
+%description    utils
+This package contains some tools and dynamic library files for libdb.
+
 %package        devel
 Summary:        Header files for libdb
 Requires:       %{name} = %{version}-%{release}
+Requires:       %{name}-utils = %{version}-%{release}
 Provides:       %{name}-devel-doc = %{version}-%{release}
 Obsoletes:      %{name}-devel-doc < %{version}-%{release}
 Provides:       %{name}-devel-static = %{version}-%{release}
@@ -181,16 +189,16 @@ mv man/* %{buildroot}%{_mandir}/man1/
 
 %files
 %defattr(-,root,root)
+%{_libdir}/libdb-*.so
+
+%files          utils
+%defattr(-,root,root)
 %doc docs/license
 %license LICENSE
 %{_bindir}/*
 %{_libdir}/*.so
 %{_datadir}/java/*.jar
-%exclude %{_libdir}/libdb.so
-%exclude %{_libdir}/libdb_cxx.so
-%exclude %{_libdir}/libdb_tcl.so
-%exclude %{_libdir}/libdb_sql.so
-%exclude %{_libdir}/libdb_java.so
+%exclude %{_libdir}/libdb-*.so
 
 %files          devel
 %defattr(-,root,root)
-- 
2.23.0

