From f7578b3f201e92618cb909508bbda7be9f51de13 Mon Sep 17 00:00:00 2001
From: yyywwwlll <yaoweili@huawei.com>
Date: Mon, 22 Mar 2021 03:26:37 +0000
Subject: [PATCH] split rpm package

---
 readline.spec | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/readline.spec b/readline.spec
index c5eba78..e48c3a2 100644
--- a/readline.spec
+++ b/readline.spec
@@ -56,15 +56,14 @@ export CPPFLAGS="-I%{_includedir}/ncurses"
 
 %install
 %make_install
-cp -a %{_libdir}/lib{readline,history}.so.7* %{buildroot}%{_libdir}
 
 %ldconfig_scriptlets
 
 %files
 %doc
 %license COPYING USAGE
-%{_libdir}/libhistory.so.*
-%{_libdir}/libreadline.so.*
+%{_libdir}/libhistory.so.8*
+%{_libdir}/libreadline.so.8*
 
 %files devel
 %{_includedir}/%{name}/*.h
-- 
2.23.0

